virttest/env_process.py

```diff
@@ -1134,6 +1134,15 @@ def postprocess(test, params, env):
     # recover the changes done to kernel params in postprocess
     kernel_extra_params_add = params.get("kernel_extra_params_add", "")
     kernel_extra_params_remove = params.get("kernel_extra_params_remove", "")
+
+    if params.get("enable_guest_iommu"):
+        enable_guest_iommu = params.get("enable_guest_iommu")
+        if enable_guest_iommu == "yes":
```

Comment:
this line is not necessary,
```python
if params.get("enable_guest_iommu") == "yes" and params.get("guest_iommu_option"):
    kernel_extra_params_add += " iommu=%s" % guest_iommu_option
```

Reply:
You code above only clean up guest_iommu_option, but not enable_guest_iommu option.
I prefer to reset all iommu-related configuration now that we add clean up step.
Thanks!

Reply:
Or it might be:
```
if params.get("enable_guest_iommu") == "yes":
    kernel_extra_params_add += " intel_iommu=on"
```
@balamuruhans What do you think?

Reply:
@vivianQizhu right! I gave an example, where we really don't need to assign a variable and again checking it.
as you have mentioned it would be good to have,
```python
if params.get("enable_guest_iommu") == "yes":
    kernel_extra_params_add += " intel_iommu=on"
    if params.get("guest_iommu_option"):
        kernel_extra_params_add += " iommu=%s" % guest_iommu_option
```

Reply:
Updated, Thanks!